src/features/missions/Missions.js

```diff
@@ -1,5 +1,31 @@
-const Missions = () => 
-  <p>missions</p>
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import TableHeader from './TableHeader';
+import TableBody from './TableBody';
+import { getMissions, joinMission, leaveMission } from './missionsSlice';
+import './mission.scss';
 
+const Missions = () => {
+  const { missions } = useSelector((state) => state).missions;
 
-export default Missions;
\ No newline at end of file
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (missions.length === 0) dispatch(getMissions());
+  }, []);
+
+  const handleMission = (id, joined) => {
+    dispatch(joined ? leaveMission(id) : joinMission(id));
+  };
+
+  return (
+    <section className="missions-section">
+      <table className="mission-table">
+        <TableHeader />
+        <TableBody onJoinMission={handleMission} missions={missions} />
+      </table>
```

Comment:
- [ ] OPTIONAL - Nice tests cases, they all pass but an error shows on the terminal: ` Warning: validateDOMNesting(...): <tbody> cannot appear as a child of <div>` Consider fixing it.